src/screens/OrganizationFunds/modal/FundModal.tsx

```
@@ -222,16 +226,16 @@ const FundModal: React.FC<InterfaceFundModal> = ({
               className={`d-flex mt-2 mb-3 flex-wrap ${mode === 'edit' ? 'justify-content-between' : 'justify-content-start gap-3'} `}
             >
               <Form.Group className="d-flex">
-                <label>{t('isTaxDeductible')} </label>
+                <label>{t('taxDeductible')} </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)